### ISSUE TITLE
multipart/form-data uploading multiple files with same field name fails

### ISSUE THREAD

**Issue by Jdruwe** [mockoon/mockoon#1613]
**Describe the bug**

Uploading multiple files using the same field name BUT not including [] returns an incorrect response

More info at: https://discord.com/channels/789493376423952394/1311280431395508225

**Steps to Reproduce**

```
public async uploadFiles(files: File[]) {
  const formData = new FormData();
  files.forEach((file) => formData.append('files', file));

  return await safeFetch(
    someZodSchema,
    `${MY_URL}/upload`,
    {
      method: 'PUT',
      headers: {
        accept: 'application/json',
      },
      body: formData,
    },
  );
}
```

In the docs (https://mockoon.com/docs/latest/templating/mockoon-request-helpers/#body) i've read: "multipart/form-data supports fields and files. Uploaded files will not be stored, but their metadata (filename, mimetype, size) will be available in the request body"

When I use below template (just for testing):

`{{body 'files'}}`

Response:

```
{
    "filename": "gladiator.jpg",
    "mimetype": "image/jpeg",
    "size": 34387,
    "": {
        "filename": "boots.jpg",
        "mimetype": "image/jpeg",
        "size": 154791
    }
}
```

This should be an array of objects containing filename, mimetype & size.

MDN:

"The append() method of the FormData interface appends a new value onto an existing key inside a FormData object, or adds the key if it does not already exist.

The difference between set() and append() is that if the specified key already exists, set() will overwrite all existing values with the new one, whereas append() will append the new value onto the end of the existing set of values." - mdn

**Mockoon version:**

v9.0.0

**OS / OS version:**

Mac OS Sonoma 14.5

**Comment by Jdruwe** [comment]
Awesome!

**Comment by 255kb** [comment]
> Awesome! Looking forward to 9.1.0, @255kb when is it planned to be released?

It should be out today or tomorrow 🙂

**Comment by 255kb** [comment]
📦This is fixed in [v9.1.0](https://mockoon.com/releases/9.1.0/).

**Comment by Jdruwe** [comment]
Upgraded and it seems to work! Thx once again
